Portability.md

```diff
@@ -39,6 +39,9 @@ characteristics:
   processes executing on the same machine.
 * An execution environment which offers forward progress guarantees to all
   threads of execution (even when executing in a non-parallel manner).
+* Availability of lock-free atomic memory operations, when naturally aligned, for
+  8- 16- and 32-bit accesses. At a minimum this must include an atomic
+  compare-and-exchange operation (including load-linked/store-conditional).
```

Comment:
"(including load-linked/store-conditional)" could be misconstrued as requiring ll/sc. How about "(load-linked/store-conditional is permitted)" or so?